kodi_voice/kodi.py

```diff
@@ -189,6 +197,16 @@ def words2roman(phrase):
     wordified = wordified + word + " "
   return wordified[:-1]
 
+def words2num(phrase):
+  wordified = ''
+  for word in phrase.split():
+    word = word.decode('utf-8')
+    try:
+      word = str(word2num(word))
+    except:
+      pass
+    wordified = wordified + word + " "
+  return wordified[:-1]
```

Comment:
That was added after I forked.  Should have checked, sorry.  I've rebased onto the latest version and switched to the proper function.